gradle-baseline-java/src/main/groovy/com/palantir/baseline/BaselineFindBugsExtension.groovy

```diff
@@ -0,0 +1,11 @@
+package com.palantir.baseline
+
+import java.util.regex.Pattern
+
+class BaselineFindBugsExtension {
```

Comment:
i think you could with `MetaClass` in groovy? but i doubt it's the right thing to do.. was just curious